src/common/transformations/src/transformations/common_optimizations/compress_float_constants.cpp

```diff
@@ -4,6 +4,8 @@
 
 #include "transformations/common_optimizations/compress_float_constants.hpp"
 
+#include <ngraph/runtime/reference/convert.hpp>
```

Comment:
@pavel-esir, try using `openvino` instead of `ngraph`, revert to `""` from `<>`.

Reply:
unfortunately it's not possible to use #include "openvino/runtime/... for reference implementation. There are no aliases introduced yet.

done `<>` -> `""`